### PR TITLE
(refactor): theming structure

### PR DESCRIPTION
**Pull request by philgetz**
Placing styles in a more logical file structure to maintain and update. Further work will follow this is mainly concerned with restructuring.

### CODE REVIEW

File: src/lib/theme/_overrides.scss [hunk 1]
```diff
@@ -0,0 +1,287 @@
...
+// All style overrides for dark, light and all components for
+// Vantage UI platform
+// 
+
+$mat-dark-theme-background: map-merge(
+  $mat-dark-theme-background,
+  (
+    card: mat-color($td-slate-dark, 1100),
+    dialog: mat-color($td-slate-dark, 600),
+    backdrop: mat-color($mat-grey, 800),
```
Comment (adamnel) [inline src/lib/theme/_overrides.scss]: What is 'backdrop'? The page background?

Comment (philgetz) [reply src/lib/theme/_overrides.scss]: The overlay over the app when the drawer is open

Comment (adamnel) [reply src/lib/theme/_overrides.scss]: On the design side we've traditionally called this 'overlay'. Will it break things for you guys to change it to that?

Comment (philgetz) [reply src/lib/theme/_overrides.scss]: Literally said "the overlay" haha, lets test it out.

File: src/lib/theme/_theme.scss [hunk 2]
```diff
@@ -4,28 +4,30 @@
...
 // Teradata Font
 $td-font-default: 'Arial, Helvetica, sans-serif';
-$td-font-regular: '"RidleyGrotesk", Arial, Helvetica, sans-serif';
-$td-font-bold: '"RidleyGrotesk", Arial, Helvetica, sans-serif';
-$td-font-semibold: '"RidleyGrotesk", Arial, Helvetica, sans-serif';
-$td-font-medium: '"RidleyGrotesk", Arial, Helvetica, sans-serif';
+$td-font-regular: '"RidleyGrotesk", "RidleyGrotesk-Regular", Arial, Helvetica, sans-serif';
+$td-font-medium: '"RidleyGrotesk", "RidleyGrotesk-Medium", Arial, Helvetica, sans-serif';
+$td-font-semibold: '"RidleyGrotesk","RidleyGrotesk-Semibold", Arial, Helvetica, sans-serif';
+$td-font-bold: '"RidleyGrotesk","RidleyGrotesk-Bold", Arial, Helvetica, sans-serif';
```
Comment (adamnel) [inline src/lib/theme/_theme.scss]: Should we name these $td-font-display-[weight] so it's clear? the difference between regular and default isn't super obvious.

Comment (philgetz) [reply src/lib/theme/_theme.scss]: done

**Comment by emoralesb05** [comment]
Really nice!

**Pull request merged**